module/os/freebsd/spl/spl_taskq.c

```diff
@@ -70,7 +70,7 @@ static unsigned long tqenthash;
 static unsigned long tqenthashlock;
 static struct sx *tqenthashtbl_lock;
 
-static uint32_t tqidnext = 1;
+static taskqid_t tqidnext;
```

Comment:
I prefer the explicit initialization for clarity, and starting at 1 avoids the extra +1.